zndraw/static/main.css

```diff
@@ -331,3 +331,15 @@ body {
   text-align: center;
   z-index: 1;
 }
+
+#ZnDrawConsoleCard {
+  width: 33%;
```

Comment:
I think we can reduce the size a little bit. On a smaller screen it feels very large, and on a large screen the text only takes half the width. Maybe 20 percent and 33 ?